Resources/translations/messages.de.yml

```diff
@@ -8,14 +8,18 @@ mb.digitizer:
     zoomTo: Heranzoomen
     edit: Bearbeiten
     attributes: Objektinformationen
-    save: Speichern
     save:
+      title: Speichern
```

Comment:
see above, maybe `caption` instead of `title`

Reply:
Anywhere else button title is called `title`. If `caption` is better to understand, then it's need to be everywhere called a `caption`. We can do that, but if we refactored all the code basis and YAML defintions